geom/multilinestring.go

```diff
@@ -0,0 +1,43 @@
+package geom
+
+import (
+	"errors"
+	osm "github.com/omniscale/go-osm"
+	"github.com/omniscale/imposm3/geom/geos"
+	"runtime"
+)
+
+func BuildMultiLinestring(rel *osm.Relation, srid int) (*geos.Geom, error) {
```

Comment:
Can you move this into geom.go? There is a multipolygon.go, but this handling is also much more complex.

Reply:
Moved.